driver-core/src/test/java/com/datastax/driver/core/querybuilder/QueryBuilderTest.java

```diff
@@ -300,6 +300,24 @@ public void updateTest() throws Exception {
     }
 
     @Test(groups = "unit")
+    public void should_handle_update_if_exists() {
```

Comment:
I trust you, will remove the failing test cases and related code in QueryBuilder then.